src/open_ai.js

```diff
@@ -0,0 +1,13 @@
+const { Configuration, OpenAIApi } = require('openai');
+
+const OPEN_AI_IMPROVE_CHANGELOG_REQUEST = `You are an expert programmer, and you are trying to rewrite release changes into user-friendly text.
+For each line bellow write one meaningful past infinitive sentence, starting each bullet point with a \`* \`, and the sentence ends with \`.\`.`;
```

Comment:
Would it understand a request to rewrite the whole changelog (or maybe each scope separately), so that we don't have to rewrite it line by line? That would be a lot faster.

Also maybe it could group similar items together (like the items about the Store ToC modal in the example).

Reply:
I tried several requests, and this one looks good so far. The text model comes with some strange ideas and comes up with things which were outside the original changes in some cases especially if you have some non-specific request like group something or sum up.

I like this one for example

```
You are an expert programmer, and you are trying to rewrite release changes into user-friendly text.
For each line below write one funny sentence, starting each bullet point with a `* `, and the sentence ends with `.`.

Resolved issue with special chars in Store ToC modal
Fixed call to actions on publication tab
Keep query string and hash when applying last used account redirect
actor run show multiple storages
paginated tables in usage breakdown
Fixed users profile data not correctly displayed in left menu
Added store ToC publication modal (and refactored code)
scheduling improvement
fix of mismatching tokens
Better copy
Better copy
Better copy
```
**Answer:**
```
- We finally fixed those pesky special characters causing problems in the Store ToC modal. No more wacky symbols!
- The call to action buttons on the publication tab are working like a charm now. Go ahead and click away!
- No more losing your place when switching accounts - the query string and hash will stay with you.
- The actor run show now has the ability to handle multiple storages, because one storage just wasn't enough.
- Paginated tables in the usage breakdown? Yes please! Scroll to your heart's content.
- The left menu now correctly displays users' profile data. No more mystery profiles.
- The store ToC publication modal is here, and the code has been tidied up for your viewing pleasure.
- Scheduling just got a whole lot better. Set your alarms and reminders with ease.
- Mismatching tokens? No more! We've fixed that issue for good.
- Our copy has gotten even better! You won't be able to resist reading it.
- You read that right, the copy is even better now. You won't be able to put it down.
- You won't believe how great the copy is now. Better grab a snack before you start reading.
```

Reply:
😄 😄 😄 This sounds like the Slack release notes